x-pack/legacy/plugins/index_management/public/sections/home/index_list/index_table/index_table.js

```diff
@@ -332,6 +347,12 @@ export class IndexTable extends Component {
                 this.toggleItem(name);
               }}
               data-test-subj="indexTableRowCheckbox"
+              aria-label={i18n.translate('xpack.idxMgmt.indexTable.selectIndexAriaLabel', {
```

Comment:
If the `title` attribute is necessary then is it worth extracting this string into a reusable constant that we can use in both places?

```js
const selectIndexAriaLabel = i18n.translate('xpack.idxMgmt.indexTable.selectIndexAriaLabel', { defaultMessage: 'Select this row' });
```

Reply:
removed the `title` attribute so this is no longer applicable